lib/standard/runners/help.rb

```diff
@@ -9,7 +9,7 @@ def call(config)
 
           Options:
 
-            --fix             Automatically fix failures where possible
+            --[no-]fix        Automatically fix failures where possible
```

Comment:
Please add `--no-fix` as a separate entry, since 99.9999% of the time people should use `--fix` and this is likely to confuse people

Reply:
Done in 62e22d268434bbd8b26fc1547108c1e337be94de